homework_4/bets_api/internal/api/controllers/controller.go

```diff
@@ -0,0 +1,90 @@
+package controllers
+
+import (
+	"github.com/gin-gonic/gin"
+	"net/http"
+)
+
+type Controller struct {
```

Comment:
The controller needs to map the bet detail to the DTO object since DTO object should not contain all data that is retrieved from the database